src/megameklab/com/ui/Aero/MainUI.java

```diff
@@ -58,30 +54,13 @@
     private BuildTab buildTab;
     private FluffTab fluffTab;
     private StatusBar statusbar;
-    JPanel masterPanel = new JPanel();
-    JScrollPane scroll = new JScrollPane();
-    private MenuBarCreator menubarcreator;
 
     public MainUI(boolean primitive) {
 
         super();
         createNewUnit(Entity.ETYPE_AERO, primitive);
         setTitle(getEntity().getChassis() + " " + getEntity().getModel() + ".blk");
-        menubarcreator = new MenuBarCreator(this);
```

Comment:
All of the stuff deleted from this point on in all of the MainUI methods is now being done in MegaMekLabMainGUI.finishSetup()